packages/realm-web/src/Fetcher.ts

```diff
@@ -64,14 +64,21 @@ function asyncIteratorFromResponseBody(body: unknown): AsyncIterable<Uint8Array>
   }
 }
 
+export type headers = { [key: string]: string };
```

Comment:
Types and interfaces should be capitalised
```suggestion
export type Headers = { [key: string]: string };
```

Reply:
On second look, found the same Headers type already declared & imported from realm-network-transport.

I chose to re-export it from Fetcher, for use in User without adding to its immediate dependencies.